services/src/openkilda-gui/ui/src/app/modules/flows/flow-detail/flow-detail.component.html

```diff
@@ -224,23 +224,30 @@ <h6 class="card-title card-custom-title">FLOW DETAILS</h6>
         </div>
         <div class="tab-pane" [ngClass]="{'active': openedTab == 'descrepancy'}">
           <div class="table-responsive">
-            <table class="table table-bordered table-hover">
-                <thead class="thead-light"><tr><th>Discrepancy</th><th>Kilda Controller</th><th>Link Store</th></tr></thead>
+            <table class="row-border hover dataTable no-footer">
+                <thead class="thead-light"><tr class="tableHeading"><th>Discrepancy</th><th>Kilda Controller</th><th>Link Store</th></tr></thead>
               <tbody>
-                <tr *ngIf="statusDescrepancy"><td>Status</td><td>{{ descrepancyData.status.controller}}</td><td>{{ descrepancyData.status.inventory}}</td></tr>
-                <tr *ngIf="bandWidthDescrepancy"><td>Bandwidth(Kbps)</td><td>{{ descrepancyData.bandwidth.controller}}</td><td>{{ descrepancyData.bandwidth.inventory}}</td></tr>
+                <tr *ngIf="statusDescrepancy" >
+                  <td class="divTableCell">Status</td>
+                  <td class="divTableCell">{{ descrepancyData.status.controller}}</td>
+                  <td class="divTableCell">{{ descrepancyData.status.inventory}}</td>
+                </tr>
+                <tr *ngIf="bandWidthDescrepancy">
+                  <td class="divTableCell">Bandwidth(Kbps)</td>
+                  <td class="divTableCell">{{ descrepancyData.bandwidth.controller}}</td>
+                  <td class="divTableCell">{{ descrepancyData.bandwidth.inventory}}</td>
+                </tr>
               </tbody>
             </table>
-            <div class="note-container">
-                <p class="col-md-6 alert alert-warning" *ngIf="(descrepancyData.status.controller =='-' && descrepancyData.status.inventory == 'Active')">
-                   <strong>*Note:</strong> Flow is not active in kilda controller</p>
-                   
-                   <p class="col-md-6 alert alert-warning" *ngIf="(descrepancyData.status.inventory == '-' ) ">
-                    <strong>*Note:</strong> Flow is not active in inventory</p>
-  
-            </div>
           </div>
-          
+          <div class="note-container">
+            <p class="col-md-6 alert alert-warning" *ngIf="(descrepancyData.status.controller =='-' && descrepancyData.status.inventory == 'Active')">
+               <strong>*Note:</strong> Flow is not active in kilda controller</p>
+               
+               <p class="col-md-6 alert alert-warning" *ngIf="(descrepancyData.status.inventory == '-' ) ">
+                <strong>*Note:</strong> Flow is not active in inventory</p>

```

Comment:
Same for inventory, if the *flow doesn't exist in the inventory* then show this message or if flow exists in inventory but not active in inventory then you can show this message.
*Flow is not active in inventory*

Reply:
resolved